game/models.py

```
@@ -363,3 +368,28 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.title} ({self.difficulty or 'Unknown'})"
+
+def _expires_at_default():
+    """Return a timestamp 48 hours from now."""
+    return timezone.now() + timedelta(hours=48)
+
+
+class GameRecord(models.Model):
```

Ruff (0.15.15)

[warning] Model does not define `__str__` method

(DJ008)